src/main/java/uk/gov/ons/census/casesvc/messaging/SurveyLaunchedReceiver.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.ons.census.casesvc.messaging;
+
+import static uk.gov.ons.census.casesvc.utility.JsonHelper.convertObjectToJson;
+
+import java.time.OffsetDateTime;
+import org.springframework.integration.annotation.MessageEndpoint;
+import org.springframework.integration.annotation.ServiceActivator;
+import org.springframework.transaction.annotation.Transactional;
+import uk.gov.ons.census.casesvc.logging.EventLogger;
+import uk.gov.ons.census.casesvc.model.dto.EventTypeDTO;
+import uk.gov.ons.census.casesvc.model.dto.ResponseManagementEvent;
+import uk.gov.ons.census.casesvc.model.entity.EventType;
+import uk.gov.ons.census.casesvc.model.entity.UacQidLink;
+import uk.gov.ons.census.casesvc.service.UacService;
+
+@MessageEndpoint
+public class SurveyLaunchedReceiver {
+
+  private final EventLogger eventLogger;
+
+  private final UacService uacService;
+
+  public SurveyLaunchedReceiver(EventLogger eventLogger, UacService uacService) {
+    this.eventLogger = eventLogger;
+    this.uacService = uacService;
+  }
+
+  @Transactional
+  @ServiceActivator(inputChannel = "surveyLaunchedInputChannel")
+  public void receiveMessage(ResponseManagementEvent event) {
+    if (event.getEvent().getType() == EventTypeDTO.SURVEY_LAUNCHED) {
```

Comment:
write this as `event.getEvent().getType() != EventTypeDTO.SURVEY_LAUNCHED` and then put the throws right after... then you don't need to have the else clause and all the business logic inside the clause, indented

Reply:
Done.